src/email/templates/ResetPassword/ResetPassword.tsx

```diff
@@ -16,7 +16,13 @@ export const ResetPassword = () => {
 			<Text>
 				You&apos;ve asked us to send you a link to reset your password.
 			</Text>
-			<Text noPaddingBottom>This link is valid for 60 minutes.</Text>
+			<Text noPaddingBottom>
+				This link is valid for 60 minutes. If your link has expired, please try{' '}
+				<a href="https://profile.theguardian.com/reset-password">
```

Comment:
I think we have a specific Link component which we prefer using instead of `<a>`.

Reply:
Thank you!